tensorflow_addons/custom_ops/activations/cc/ops/rrelu_op.cc

```diff
@@ -0,0 +1,45 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#include "tensorflow/core/framework/common_shape_fns.h"
+#include "tensorflow/core/framework/op.h"
+#include "tensorflow/core/framework/shape_inference.h"
+
+namespace tensorflow {
+namespace addons {
+
+REGISTER_OP("Addons>Rrelu")
+    .Input("features: T")
+    .Output("activations: T")
+    .Output("alpha: T")
+    .Attr("T: {half, float, double}")
+    .Attr("lower: float")
```

Comment:
```
lower: float >=0
upper: float <1
```
Throw an unparse error.Why?

Reply:
Seems that only `list` and `int` support this syntax.
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/framework/op_def_builder.cc#L250